src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_CATEGORY = "supply";
+    private static final String BUY_CATEGORY = "buy";
+    private static final String SPLIT_VALUE = ",";
+    private static final String RESULT_CATEGORY = "result";
+    private static final int FIRST_VALUE = 0;
+    private static final int SECOND_VALUE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeInFile(getResult(readFile(fromFileName)), toFileName);
+    }
+
+    public String getResult(StringBuilder readFile) {
```

Comment:
let's not pass StringBuilder as an argument or return from method. It's generally considered as bad practice